presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestPrestoDriverUri.java

```diff
@@ -235,6 +238,16 @@ public void testUriWithExtraCredentials()
         assertEquals(properties.getProperty(EXTRA_CREDENTIALS.getKey()), extraCredentials);
     }
 
+    @Test
+    public void testUriWithSessionProperties()
+            throws SQLException
+    {
+        String sessionProperties = "sessionProp1:sessionValue1;sessionProp2:sessionValue2";
+        PrestoDriverUri parameters = createDriverUri("presto://localhost:8080?extraCredentials=" + sessionProperties);
+        Properties properties = parameters.getProperties();
+        assertEquals(properties.getProperty(EXTRA_CREDENTIALS.getKey()), sessionProperties);
```

Comment:
we should either use SESSION_PROPERTIES.getKey or rename the test that tests extra credentials. And have another test for SESSION_PROPERTIES to make sure if behavior changes for SESSION_PROPERTIES, test breaks and need to be fixed.

Reply:
Thanks, you are correct.